src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -98,10 +98,22 @@ public void keyboardShortcutsTest() {
 
         click("#dummy/dummy_col1_1");
 
-        // mark as read/unread
+        // mark as read
         ListPanel issuePanel = find("#dummy/dummy_col1");
+        push(KeyCode.HOME);
+        int issueIdBeforeMark = selectedIssueId;
+        int issueIdExpected = issueIdBeforeMark - 1;
         push(getKeyCode("MARK_AS_READ"));
+        assertEquals(issueIdExpected, selectedIssueId);
+        push(getKeyCode("UP_ISSUE")); //required since focus has changed to next issue
+        assertEquals(true, issuePanel.getSelectedIssue().isCurrentlyRead());
+        
+        push(KeyCode.END);
+        push(getKeyCode("MARK_AS_READ"));
+        assertEquals(1, selectedIssueId);
         assertEquals(true, issuePanel.getSelectedIssue().isCurrentlyRead());
+        
```

Comment:
1. I wouldn't mind a bit more comments in tests, to explain the intention of each test case, what is expected etc. 
2. Tests can be more extensive. e.g. Corner cases such as there is only one issue in the list, no issues in the list, etc.